src/test/java/callhub/connect/controllers/FileControllerTests.java

```diff
@@ -120,4 +125,58 @@ void testFindDocumentByID_InvalidID() throws Exception {
         RequestBuilder request = get("/files/12345678");
         mockMvc.perform(request).andExpect(status().isBadRequest()).andExpect(content().string("File could not be found."));
     }
+
+    @Test
+    void testPDFLinkValidSession() throws Exception {
+        String currentDirectory = System.getProperty("user.dir");
+        String pathName = currentDirectory + "/src/main/java/callhub/connect/pdfs/fall2023.pdf";
+        File pdfFile = new File(pathName);
+        MockMultipartFile sampleFile = new MockMultipartFile(
```

Comment:
Duplication in Test Setup: 
- There is some duplication in setting up the `MockMultipartFile` in both test methods. 
- Consider refactoring this into a separate method or a @BeforeEach setup method to reduce duplication!

Reply:
remind me to fix this after i merge